awx/ui_next/src/screens/Credential/shared/CredentialForm.jsx

```diff
@@ -108,6 +118,8 @@ function CredentialFormFields({
         value={orgField.value}
         touched={orgMeta.touched}
         error={orgMeta.error}
+        required={verifyGalaxyCredential(credTypeField.value)}
```

Comment:
I suppose you could just store this value in a variable instead of calling a function (since you have to do it twice).

```
  const isGalaxyCredential =
    !!credTypeField.value &&
    credentialTypes[credTypeField.value].kind === 'galaxy'
      ? true
      : false;

...

      <OrganizationLookup
        helperTextInvalid={orgMeta.error}
        isValid={!orgMeta.touched || !orgMeta.error}
        onBlur={() => orgHelpers.setTouched()}
        onChange={onOrganizationChange}
        value={orgField.value}
        touched={orgMeta.touched}
        error={orgMeta.error}
        required={isGalaxyCredential}
        autoPopulate={isGalaxyCredential}
      />
```